src/components/MobileMenu/MobileMenu.jsx

```diff
@@ -0,0 +1,97 @@
+import "./MobileMenu.css";
+import logo from "../../assets/SSH_logo.png";
+import { useContext } from "react";
+import { Link } from "react-router-dom";
+import logout from "../../assets/logout.png";
+import logout_dark from "../../assets/logout-dark.png";
+
+import CurrentUserContext from "../../contexts/CurrentUserContext.js";
+
+function MobileMenu({
+  isMenuOpen,
+  handleCloseMobileMenu,
+  handleSignUpClick,
+  handleLogInClick,
+  handleLogOut,
+  handleAddArticleClick,
+  inProfile,
+}) {
+  const { currentUser, isLoggedIn } = useContext(CurrentUserContext);
+
+  return (
+    <div className={`mobile_menu ${isMenuOpen ? "mobile_menu_open" : ""}`}>
```

Comment:
`mobile_menu` 

There is a BEM mistake


It's a modificator of `mobile`, but you don't have such a block


You should use `mobile-menu` because the other class names are **elements** of this block in your code